test_unstructured/documents/test_elements.py

```diff
@@ -66,13 +65,10 @@ def test_text_element_apply_cleaners():
 
 
 def test_text_element_apply_multiple_cleaners():
-    cleaners = [
-        partial(clean_prefix, pattern=r"\[\d{1,2}\]"),
-        partial(translate_text, target_lang="ru"),
-    ]
-    text_element = Text(text="[1] A Textbook on Crocodile Habitats")
+    cleaners = [partial(clean_prefix, pattern=r"\[\d{1,2}\]"), partial(clean_bullets)]
+    text_element = Text(text="[1] \u2022 A Textbook on Crocodile Habitats")
```

Comment:
This test was running slow, hanging for several seconds on every test run, because it invokes a translation model. Since translation is not the behavior under test here (working with multiple cleaners is), choose a different cleaner that does not incur that excessive overhead.